### PR TITLE
Common UAVCAN: f303-universal & rangefinder instructions

### PR DESCRIPTION
**Pull request by ukmook**
Added instructions for f303-universal firmware and rangefinder setup for AP_Periph 1.1.0

### CODE REVIEW

File: common/source/docs/common-uavcan-adapter-node.rst [hunk 1]
```diff
@@ -60,8 +60,36 @@ An AP_Periph UAVCAN firmware supports these UAVCAN features:
...
 ======
 
-Rangefinder
-------------
+AP_Periph f303-universal Firmware
+---------------------------------
+
+ The f303-Universal Firmware has the ability to be used for several devices but only one can be
+ enabled to use the single uart.
+ Once uploaded the default device is set to GPS, to use another device such as Rangefinder the
```
Comment (Hwurzburg) [inline common/source/docs/common-uavcan-adapter-node.rst:68]: Once uploaded the default device to be connected to the UART port is set to GPS....

File: common/source/docs/common-uavcan-adapter-node.rst [hunk 2]
```diff
@@ -60,8 +60,36 @@ An AP_Periph UAVCAN firmware supports these UAVCAN features:
...
+
+  The Firmware can also be used for I2C peripherals.
+
+ - COMPASS
+ - BARO
+ - AIRSPEED SENSOR
+ - NCP5623 LED
+
+ Rangefinder
+ ------------
```
Comment (Hwurzburg) [inline common/source/docs/common-uavcan-adapter-node.rst:92]: underlines must be the same length as text above

File: common/source/docs/common-uavcan-adapter-node.rst [hunk 3]
```diff
@@ -60,8 +60,36 @@ An AP_Periph UAVCAN firmware supports these UAVCAN features:
...
-Rangefinder
-------------
+AP_Periph f303-universal Firmware
+---------------------------------
+
+ The f303-Universal Firmware has the ability to be used for several devices but only one can be
+ enabled to use the single uart.
+ Once uploaded the default device is set to GPS, to use another device such as Rangefinder the
+ GPS has to be turned off and Rangefinder or other device enabled. To achieve this either use
+ UAVCAN GUI or MissionPlanner SLCAN. (link here). Once connected set params i.e. for TFmini_S/Plus rangefinder
```
Comment (Hwurzburg) [inline common/source/docs/common-uavcan-adapter-node.rst:70]: use UAVCAN GUI or Mission Planner SLCAN to set the UAVCAN adpater's internal parameters.

fyi https://uavcan.org/GUI_Tool/Overview/ is UAVCAN GUI link
ref: for MP SLCAN wiki page is <common-mp-slcan>

Comment (ukmook) [reply common/source/docs/common-uavcan-adapter-node.rst:70]: Thanks for the fyi, I was unsure of the correct usage

**Comment by Hwurzburg** [comment]
I need to work on this page a bit to make f103 and f303 distinctions..will do tomorrow, but merging as is...

**Pull request merged**